src/ebusd/bushandler.cpp

```diff
@@ -438,6 +438,7 @@ result_t BusHandler::handleSymbol() {
 
   case bs_skip:
     timeout = SYN_TIMEOUT;
+    [[fallthrough]];
```

Comment:
is this compliant to older GCC versions?

Reply:
As long as GCC >=  7 is true, then the answer is yes.
If support for even older versions is required, this might be an option:

```
#ifndef FALLTHROUGH
#if defined(__GNUC__) && __GNUC__ >= 7
#define FALLTHROUGH [[fallthrough]]
#else
#define FALLTHROUGH do {} while (false)
#endif
#endif
```